### ISSUE TITLE
[PHP] Add Project Status table for each Signal in README top section

### ISSUE THREAD

**Issue by trivikr** [open-telemetry/opentelemetry-php#766]
**Is your feature request related to a problem? Please describe.**

The Project Status of Traces/Metrics/Logs Signals in OpenTelemetry is difficult to find in OpenTelemetry for ~~Java~~ PHP.

**Describe the solution you'd like**
Add Project Status Table at the top of README, similar to how it's done in other implementations:

- .NET https://github.com/open-telemetry/opentelemetry-dotnet
- Go https://github.com/open-telemetry/opentelemetry-go
- CPP https://github.com/open-telemetry/opentelemetry-cpp

The Status Table would list Signal and the current status

```
| Signal  | Status                            | Project                           |
| ------- | --------------------------------- | --------------------------------- |
| Traces  | Experimental/Alpha/Beta/RC/Stable | [version](link-to-github-project) |
| Metrics | Experimental/Alpha/Beta/RC/Stable | [version](link-to-github-project) |
| Logs    | Experimental/Alpha/Beta/RC/Stable | [version](link-to-github-project) |
```

**Describe alternatives you've considered**
Going through repo contents to find the status of each Signal.

**Additional context**
Summary table for OpenTelemetry Status https://github.com/open-telemetry/opentelemetry.io/issues/1548

**Comment by trivikr** [comment]
On a quick look at version numbers and GitHub projects, this is what I think the table is:

| Signal  | Status       | Project |
| ------- | ------------ | ------- |
| Traces  | Experimental | N/A     |
| Metrics | Experimental | N/A     |
| Logs    | N/A          | N/A     |

If it's incorrect, please provide the correct table in this GitHub issue, so that it can be updated on README.

**Comment by brettmc** [comment]
@trivikr - the table looks correct to me. Did you want this assigned to you?

**Comment by tidal** [comment]
> The Project Status of Traces/Metrics/Logs Signals in OpenTelemetry is difficult to find in OpenTelemetry for Java.

-> https://github.com/open-telemetry/opentelemetry-java

**Comment by trivikr** [comment]
I'd use issue with OpenTelemetry Java as a reference while copying requests. Updated it to use PHP.

**Comment by trivikr** [comment]
The consensus is being achieved towards treating website as authoritative, and provide a link to website instead. 

https://github.com/open-telemetry/opentelemetry-java/pull/4612#issuecomment-1185118267

We'll request the status page to link to language instrumentation links instead of GitHub READMEs in https://github.com/open-telemetry/opentelemetry.io/issues/1548

**Comment by tidal** [comment]
> The consensus is being achieved towards treating website as authoritative, and provide a link to website instead.

That's certainly not a consensus otherwise this sentence would be obsolete. If you are talking about the website repo, it's not authoritative in our case, as the truth about this libarary and its content is this repository which will not simply change because of any content on the website.

**Comment by brettmc** [comment]
Implemented in #816
